src/test/scala/uk/gov/hmrc/perftests/itsass/RequestsHelper.scala

```diff
@@ -18,15 +18,21 @@ package uk.gov.hmrc.perftests.itsass
 
 import io.gatling.core.Predef._
 import io.gatling.core.check.CheckBuilder
+import io.gatling.core.check.css.CssCheckType
 import io.gatling.core.check.regex.RegexCheckType
+import io.gatling.core.structure.ChainBuilder
 import io.gatling.http.check.header.HttpHeaderRegexCheckType
 import io.gatling.http.Predef._
+import io.gatling.http.check.HttpCheck
```

Comment:
did you mean to import all these? they're all unused

Reply:
done